client/go/outline/electron/main.go

```diff
@@ -97,6 +101,15 @@ func main() {
 		os.Exit(exitCodeSuccess)
 	}
 
+	if *args.fetchConfig != "" {
```

Comment:
This feels a bit awkward to wedge it into this app, but there's probably a good reason for doing it all with 1 app. We should at least update the docstring for this main to reflect the different uses?

Reply:
Added comment to the go-doc of main.